ui/src/App.vue

```diff
@@ -41,19 +44,31 @@ watch(
 <template>
   <!-- Force LTR/RTL on whole app, we do not use vuetify locales at all -->
   <v-locale-provider :rtl="selectedLanguage.rtl">
-    <div class="text-center logo-div">
+    <div class="text-center" :class="smAndDown ? 'logo-div-sm' : 'logo-div'">
       <a :href="config?.home_page" target="_blank">
         <img src="./assets/ZIMIT_LOGO_RGB.svg" class="logo-img vue" alt="Vue logo" />
       </a>
     </div>
-    <v-select
-      v-model="selectedLanguageItem"
-      class="language-select"
-      :items="languageItems"
-      density="compact"
-      return-object
-    >
-    </v-select>
+    <div class="top-corner">
+      <v-btn
+        class="donate-btn btn-green-rev"
+        append-icon="fa-gift"
+        variant="outlined"
+        size="large"
+        href="https://kiwix.org/en/get-involved/#donate"
```

Comment:
I also think it's better to send them to our website. The donoborbox JS sends people to `https://donorbox.org/kiwix?default_interval=m` instead.